test/integration/auth.spec.ts

```diff
@@ -2132,12 +2232,11 @@ function safeDelete(uid: string): Promise<void> {
  * @param {string[]} uids The list of user identifiers to delete.
  * @return {Promise} A promise that resolves when delete operation resolves.
  */
-function deleteUsersWithDelay(uids: string[]): Promise<admin.auth.DeleteUsersResult> {
-  return new Promise((resolve) => {
-    setTimeout(resolve, 1000);
-  }).then(() => {
-    return admin.auth().deleteUsers(uids);
-  });
+async function deleteUsersWithDelay(uids: string[]): Promise<admin.auth.DeleteUsersResult> {
+  if (!authEmulatorHost) {
```

Comment:
Nit: true condition first

Reply:
I don't see how to flip this since the delay is only needed if NOT running against emulator.

Reply:
Ah, ignore my comment. I misread the code.